skins/cat17/use-of-funds/src/components/FundSection.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-	<div id="fund-section">
+	<div class="fund-section">
```

Comment:
Hey, thanks for your late-night review! I have to admit, I don't fully get this. This seems to produce invalid HTML because it will create multiple divs with the same ID which is not valid, as you said yourself:

![ids](https://user-images.githubusercontent.com/4571512/48165443-e4464a00-e2e5-11e8-8d2f-2c8efb131e98.png)

Do you have more infos on this that I can read up on? Not saying this is wrong but this seems really weird to me.